tests/e2e/test_rosacli_cluster.go

```diff
@@ -21,8 +21,7 @@ import (
 )
 
 var _ = Describe("Edit cluster",
-	labels.Day2,
```

Comment:
Why not keeping the `lables.Runtime.Day2` here ? All edit are day2 operations

Reply:
There may be potential for different runtime on cases. I'd like to set the `runtime` labels on the case level so we keep the uniform rule.

Reply:
Ok. Please add a line to the README to set the Runtime label on the case